src/com/perl5/lang/perl/util/PerlRunUtil.java

```diff
@@ -65,6 +65,12 @@ public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, @N
 	@NotNull
 	public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, @NotNull String perlDirectory, @Nullable VirtualFile scriptFile, String... perlParameters)
 	{
+		return getPerlCommandLine(project, Collections.emptyList(), perlDirectory, scriptFile, perlParameters);
+	}
+
+	@NotNull
+	public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, List<String> additionalLibraries, @NotNull String perlDirectory, @Nullable VirtualFile scriptFile, String... perlParameters)
```

Comment:
There is no need to add an `additionalLibraries` parameter. You already have `perlParameters`. You may add them using `com.perl5.lang.perl.idea.run.PerlRunProfileState#getPerlArguments` or make a separate method to create an array of additional -I args and merge them with `ArrayUtil` for example.